docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
+
+* **TUESDAY**: The Job Fair will be Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon. 
+
+See the :doc:`full schedule </conf/{{ shortcode }}/{{ year }}/schedule>` for exact timing details.
+
+Sponsorship platform
+--------------------
+
+We will be using `Hopin <https://hopin.to/>`_ as our online conference platform. It allows a number of unique spaces for attendees during the conference, and we hope it will allow a good amount of interaction between attendees and sponsors.
+
+**We will open the online platform 1 week prior to the conference starting**. At that point, you will be able to login and look around the platform, but it won't become fully active until the Sunday of the conference.
+
+Hopin has a set of `sponsor information <https://support.hopin.to/en/articles/3574463-vendor-instructions>`_ that explains a bit more about the sponsorship setup on the platform.
+
+Sponsorship spaces
+------------------
+
+A quick overview of the spaces in the venue that are important:
+
+* The *main stage* is where the talks happen. This is also where Lightning talks will be given.
+* The *session area* is where the Unconference & Job Fair will happen. Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference.
+* The *expo area* is where sponsors will be able to have booths available. They are able to provide videos that will play 
+* The *networking area* is where you can chat with a random attendee to get to know people. There is not specific sponsorship usage of this area.
+
+Sponsorship events
+------------------
+
+Job Fair
+~~~~~~~~
+
+On Tuesday morning we hold our Job Fair,
+which is a wonderful place to connect with our over 500 attendees.
+Many of them are looking for jobs now or will be in the near future,
+so it's a great chance to talk more about your company culture and open positions.
+
+**Logistics**: You will be assigned a session in the *session area* where you can engage with attendees and answer questions. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people. 
+
+Sponsor booths
+~~~~~~~~~~~~~~
+
+Sponsor booths are only available for our larger sponsors.
+You will be given a booth that is in the *expo area* of the online platform.
+Most of the attendee focus will come during breaks between talks and at the beginning and end of the day.  
+
+**Logistics**: You can record a video to play at your booth when it isn't staffed, but we generally recommend having someone with the expo hall tab open to answer any questions that attendees might have. We recommend that you answer answer general questions in the main room and then break off into private calls or chat to talk in more depth to specific people. 
```

Comment:
```suggestion
**Logistics**: You can record a video to play at your booth when attendees enter the booth, but we generally recommend having someone with the Expo hall tab open to answer any questions that attendees might have. We also recommend that you answer general questions in the main booth and then break off into private calls or chat to talk in more depth to specific people. 
```

Reply:
How do we plan to request booth information from sponsors? We need to populate the fields in the Expo instance in hopin before the conference starts. I can make a google form if you need help with that...